pos_customer_required/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# Copyright (C) 2004-Today Apertoso NV (<http://www.apertoso.be>)
+# Copyright (C) 2016-Today: La Louve (<http://www.lalouve.net/>)
+# Copyright (C) 2019-Today: Druidoo (<https://www.druidoo.io>)
+# @author: Jos DE GRAEVE (<Jos.DeGraeve@apertoso.be>)
+# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html
+
+{
+    "name": "Point of Sale Require Customer",
+    "version": "13.0.1.0.0",
+    "category": "Point Of Sale",
+    "summary": "Point of Sale Require Customer",
+    "author": "Apertoso NV, La Louve, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/pos",
+    "license": "AGPL-3",
+    "depends": ["point_of_sale"],
+    "data": [
+        "static/src/xml/templates.xml",
```

Comment:
The location of this file is misleading.
`static/src` is meant for content served statically to the frontend, not `ir.ui.view` definitions.

Could you please move this file to:
```suggestion
        "views/assets.xml",
```